pkgs/top-level/perl-packages.nix

```diff
@@ -28609,6 +28594,35 @@ with self; {
     };
   };
 
+  Xapian = buildPerlModule rec {
+    pname = "Xapian";
+    version = "1.4.27";
+    src = fetchurl {
+      url = "https://oligarchy.co.uk/xapian/${version}/xapian-bindings-${version}.tar.xz";
+      sha256 = "1fhq6rydjymmyn79cdza0j4rmlizrrwmf5mx276rlmwyh085wfxs";
+    };
+    buildInputs = [ pkgs.xapian ];
+    preConfigure = ''
+      # FIXME: doesn't work for cross
+      export PERL_LIB="$out/lib/perl5/site_perl/${perl.version}"
+      export PERL_ARCH="$PERL_LIB/$(perl -MConfig -e 'print $Config{archname}')"
+    '';
+    configureFlags = [
+      "--with-perl"
+    ];
+    outputs = [ "out" ]; # no "devdoc"
+    # Use default phases
+    buildPhase = null;
```

Comment:
I can find only one occurrence of `buildPhase *= *null` in Nixpkgs, looks like `dontBuild = true` is more idiomatic.

Reply:
Setting `buildPhase = null` isn't the same as `dontBuild = true` as the comment above states.

Reply:
oh sorry, I missed that comment in Github's view, thanks, I didn't know we could do that.